explorer/client/src/utils/internetapi/SuiRpcClient.ts

```diff
@@ -92,17 +84,19 @@ type BoolString = 'true' | 'false';
 export type JsonBytes = { bytes: number[] };
 export type MoveVec<T extends object | string> = { vec: T[] };
 
-export interface ObjectInfoResponse<T> {
+export interface ObjectInfoResponse {
     owner: string;
     version: string;
     id: string;
     readonly: BoolString;
     objType: string;
-    data: SuiObject<T>;
+    data: SuiObject;
 }
 
-export interface SuiObject<T> {
```

Comment:
this generic parameter is to allow more specifically typed responses , beyond 
```typescript
contents: {
        [key: string]: any;
 }
 ```
 
 it should be possible to keep it and have everything work (by using `{ [key: string]: any }` as `T`)

Reply:
Implemented